Customers.cs

```diff
@@ -20,34 +20,39 @@ public Customers()
             InitializeComponent();
         }
 
+       //SqlConnection connection = new SqlConnection(ConfigurationManager.ConnectionStrings["hotel.Properties.Settings.Setting"].ConnectionString);
+       
         
-       SqlConnection connection = new SqlConnection(ConfigurationManager.ConnectionStrings["hotel.Properties.Settings.Setting"].ConnectionString);
-      
+        private string ConnectionString = "hotel.Properties.Settings.Setting";
```

Comment:
Bir nokta daha :) Buraya gercek connection string'i koy. Yani asagidaki `ConfigurationManager...` kismini buraya koy. Her ne kadar maliyetli bir islem olmasa da ayni seyleri surekli yapmanin anlami yok. Bir kere connection string'i alip onu degiskende tutarsan bir miktar daha az is yaptirmis olursun. Ileride daha maliyetli isler icin de aliskanlik kazanmis olursun.

Reply:
Teşekkür ederimm.
Tüm metodlar için tamamladım.